### ISSUE TITLE
"An error has occured when trying to reach the server. Please go back to home screen and retry."

### ISSUE THREAD

**Issue by nicolas-raoul** [WikiJourney/wikijourney_app#99]
Request to http://api.wikijourney.eu/?place=paris&maxPOI=25&range=1.0&lg=en fails.
It seems like wikijourney.eu is down.

**Comment by sylvainar** [comment]
That's right, our website and API are down. We're currently investigating, thanks for reporting.

**Comment by nymous** [comment]
We moved the website and the API to our backup server (free web host), so the app is working again.
This is still a very brittle setup, so we should think of a better way to support this.
